### DISCUSSION TITLE
Motion recording not working.

### DISCUSSION THREAD

**Discussion by tripsylon** [bangcorrupt/hacktribe#284] [Q&A] [answered]
Hello, after the latest update the motion recording isn’t working on numerous projects, even after saving the project on another pattern number and after rebooting the machine… I’m using the red ES2. Does someone have a solution? Thank you.

**Comment by bangcorrupt** [comment]
I can look into this later.  Some questions:

Are you using the `main` branch?

Does motion recording work from a freshly initialised pattern?

Does motion recording work if you clear motion first (also clear MFX motion) ?

Can you share a pattern file that won't record motion (and a pattern file that will)?

**Reply by tripsylon** [reply]
I found out something interesting a minute ago,on all the not working projects I had the new tap delay MFX on, after switching the mfx three fx up or two fx down the issue was repaired even after switching back to the tap delay, but sometimes it comes back…

I’m not sure what you mean by main branch… I installed the version from the Hacktribe tutorial you shared me last week 

No fresh patterns are fine  it only appears after some time maybe after the MFX insertion but I’m not sure… 

Clearing the motions did not help… 

I will share the files later today . 

Thank you for your help 🙏😌

**Reply by tripsylon** [reply]
It’s the tap delay it happens on all projects on which I insert it.

**Reply by bangcorrupt** [reply, marked as answer]
Confirmed bug, thank you for testing.

#283
